pysatSpaceWeather/instruments/methods/swpc.py

```diff
@@ -962,3 +969,43 @@ def list_files(name, tag, inst_id, data_path, format_str=None):
         files.loc[files.index[-1] + pds_offset] = files.values[-1]
 
     return files
+
+
+def find_issue_date(file_txt, date_fmt="%H%M UT %d %b %Y"):
```

Comment:
I'd recommend adding tests for this new function.

Reply:
Added #150 to address this.